dependency-check-suppressions.xml

```diff
@@ -26,7 +22,23 @@
   </suppress>
 
   <suppress>
-    <notes>False-positive. See CCD-3252</notes>
```

Comment:
Can you please add the JIRA ticket number for CVE-2022-22968?

Reply:
I don't see CVE-2022-22968 in the suppression file for ccd-test-stubs-service